cg/meta/upload/scout/files.py

```diff
@@ -0,0 +1,188 @@
+        """Include all files that are used on case level in scout"""
+        raise NotImplementedError
+
+    def include_multiqc_report(self, case: ScoutLoadConfig) -> None:
+        LOG.info("Include MultiQC report to case")
+        case.multiqc = self.fetch_file_from_hk(self.case_tags.multiqc_report.hk_tags)
+
+    def include_delivery_report(self, case: ScoutLoadConfig) -> None:
+        LOG.info("Include delivery report to case")
+        case.delivery_report = self.fetch_file_from_hk(self.case_tags.delivery_report.hk_tags)
+
+    def include_sample_alignment_file(self, sample: ScoutIndividual) -> None:
+        """Include the alignment file for a sample
+
+        First add the bam.
+        Cram is preferred so overwrite if found
+        """
+        sample_id: str = sample.sample_id
+        bam_tag_obj: TagInfo = self.sample_tags.bam_file
+        sample.alignment_path = self.fetch_sample_file(tag_map=bam_tag_obj, sample_id=sample_id)
+
+        cram_tag_obj: TagInfo = self.sample_tags.alignment_file
+        sample.alignment_path = self.fetch_sample_file(tag_map=cram_tag_obj, sample_id=sample_id)
+
+    def fetch_sample_file(self, tag_map: TagInfo, sample_id: str) -> Optional[str]:
+        """Fetch a file that is specific for a individual from housekeeper"""
+        tags: set = tag_map.hk_tags.copy()
+        tags.add(sample_id)
+        return self.fetch_file_from_hk(hk_tags=tags)
+
+    def fetch_file_from_hk(self, hk_tags: Set[str]) -> Optional[str]:
+        """Fetch a file from housekeeper and return the path as a string.
+        If file does not exist return None
+        """
+        hk_file: Optional[hk_models.File] = HousekeeperAPI.fetch_file_from_version(
+            version_obj=self.hk_version_obj, tags=hk_tags
+        )
+        if hk_file is None:
+            return hk_file
+        return hk_file.full_path
+
+
+class BalsamicFileHandler(ScoutFileHandler):
+    def __init__(self, hk_version_obj: hk_models.Version):
+        super().__init__(hk_version_obj=hk_version_obj)
+        self.case_tags: BalsamicCaseTags = BalsamicCaseTags()
+        self.sample_tags: BalsamicSampleTags = BalsamicSampleTags()
+
+    def include_case_files(self, case: BalsamicLoadConfig):
+        LOG.info("Including BALSAMIC specific case level files")
+        case.vcf_cancer = self.fetch_file_from_hk(self.case_tags.snv_vcf.hk_tags)
+        case.vcf_cancer_sv = self.fetch_file_from_hk(self.case_tags.sv_vcf.hk_tags)
+        self.include_multiqc_report(case)
+
+    def include_sample_files(self, sample: ScoutBalsamicIndividual):
+        LOG.info("Including BALSAMIC specific sample level files")
+
+
+class MipFileHandler(ScoutFileHandler):
+    def __init__(self, hk_version_obj: hk_models.Version):
+        super().__init__(hk_version_obj=hk_version_obj)
+        self.case_tags = MipCaseTags()
+        self.sample_tags = MipSampleTags()
+
+    def include_case_files(self, case: MipLoadConfig):
```

Comment:
can "case" be changed to comething more intuitive/discriptive? Ypu are referring to scout config model, som maybe something along those lines?